README.md

```diff
@@ -225,6 +225,15 @@ suspend fun resumeWebSocketConnection(): Result<Boolean>
 
 --------------------
 
+#### `ChatSession.reset`
+Resets the ChatSession object which will disconnect the webSocket and remove all session related data without disconnecting the participant from the chat contact.
```

Comment:
Can you mention here as well that the chat will not be ended?

Reply:
When saying "the chat will not be ended", doesn't that just mean that we aren't disconnecting the participant? 

If anything, on the client side, the chat session has ended since we've reset all relevant chat session data.  And whether the chat contact ends wouldn't be determined on the client side but rather the contact flow side.